manifests/server/config.pp

```diff
@@ -383,4 +387,26 @@
       require    => Openldap['cn=module{0},cn=config'],
     }
   }
+
+  if $::openldap::server::ppolicy {
+
+    openldap { "olcOverlay=${overlay_index['ppolicy']},olcDatabase={${db_index}}${db_backend},cn=config": # lint:ignore:80chars
+      ensure     => present,
+      attributes => delete_undef_values({
+        'objectClass'              => [
+          'olcOverlayConfig',
+          'olcPPolicyConfig',
+        ],
+        'olcOverlay'               => 'ppolicy',
+        'olcPPolicyDefault'        => "cn=passwordDefault,${::openldap::server::suffix}",# lint:ignore:80chars
```

Comment:
I'm going to add a function that checks a DN is either the same or a subtree of another DN, i.e. the base DN of the directory. That can then be used to validate that this default policy object would exist somewhere within the directory but allow the user free reign to put it wherever they want. It's not a completely unreasonable default but I'd personally put it somewhere else.